### ISSUE TITLE
Grunt 'compress' task is broken

### ISSUE THREAD

**Issue by alexesprit** [david-sabata/web-scrobbler#1088]
The target archive doesn't contain all needed files (e.g. `core/**`, `dialogs/**` etc). So in this time the task is completely useless.

Should the task be removed?

**Comment by inverse** [comment]
Would it be better to fix the grunt task? @david-sabata did you use this when deploying?

**Comment by david-sabata** [comment]
I did, but I don't anymore. Probably can be removed, as it's not needed until we have some automated release process (=never :grinning: )

**Comment by inverse** [comment]
That's a shame :( would be pretty sweet to have automagic builds done from tagged releases!

**Comment by alexesprit** [comment]
I was looking for ways to automate releases using Github webhooks and I found a [script](https://github.com/robertknight/passcards/blob/master/utils/publish-chrome-extension.js) which can be used to upload a packed extension to Chrome Web store (unfoternately, I cannot check it; so the script can be broken); a [library](https://github.com/nlf/node-github-hook) for easy setup of webhook. But this approach requires a server that will listen for incoming event from Github, pack the extension and upload it. This way was originally suggested by @david-sabata in the Gitter chat.

I also noticed Travis CI has a 'deploy' feature. Seems like it can be configured to call the script mentioned above on pushing tags. It differs from previous way, so we probably should add some checks to get it properly worked.
